src/webgl/program.js

```diff
@@ -94,14 +94,14 @@ export default class Program extends Resource {
   // that have sane defaults.
   draw({
     drawMode = GL.TRIANGLES,
-    vertexCount,
+    vertexCount = undefined,
```

Comment:
This may trigger linter warnings as it is `undefined` by default.  FWIW, I often use `null` to show the next programmer that something was explicitly set to an invalid value as opposed to just left `undefined` by mistake.